src/base/types/MetabolicModel.jl

```diff
@@ -314,6 +313,17 @@ function gene_notes(model::MetabolicModel, gene_id::String)::Notes
     return Dict()
 end
 
+"""
+    enzyme_capacity(model::MetabolicModel)
+
+Return enzyme capacity inequality constraint vector and bound, or nothing
+if it doesn't exist in the model.
+"""
+function enzyme_capacity(model::MetabolicModel)
+    #TODO this needs a type
+    nothing, nothing
+end
+
```

Comment:
`enzyme_capacity` is not really used anywhere as an accessor, right? IMO can be removed

(we might recreate later though)

Reply:
yep, this can be deleted, everything is now wrapped as a coupling